--- upstream/Figure4/03-Plot_UniVariate_METABRIC_TCGA.R:106-136
+++ patched/supp6c_uni_tcga_os.R
@@ -1,8 +1,8 @@
-## TCGA - Overall Survival
-TCGA_OS_Univariate <- read.table(paste0(args$TCGA_dir, "/TCGA_OS_CoxUniVariate.tsv"),
+## TCGA_LUAD - Overall Survival
+TCGA_LUAD_OS_Univariate <- read.table(paste0(args$TCGA_LUAD_dir, "/TCGA_LUAD_OS_CoxUniVariate.tsv"),
     header=TRUE, sep='\t')
 
-TCGA_OS_Univariate_plt <- TCGA_OS_Univariate %>% 
+TCGA_LUAD_OS_Univariate_plt <- TCGA_LUAD_OS_Univariate %>% 
     mutate(Point_Type = case_when(
         LogRank_PVal >= 0.05 ~ "Not Significant",
         log2(Hazard_Ratio) > 0 ~ "Risk",
@@ -26,6 +26,6 @@
     labs(x = "Log2(Hazard Ratio)",
          y = "-Log10(CoxUnivPval)") +
     annotate("text", x = -Inf, y = -Inf, 
-             label = "OS\nTCGA", 
+             label = "OS\nTCGA_LUAD", 
              hjust = 0, vjust=-0.5, 
              size = 5)
